english/forest/shelter/play-dead/get-out/get-out.md

```diff
@@ -0,0 +1,18 @@
+After leaving the improvised tent, you realize that it's been raining for a while
+ now. The footprints of the bear are clearly visible in the wet terrain.
+
+
+You make a few steps further, when suddenly a lightning strikes your little shelter!
+
+Shreds of wood and plants are blasted away and your parachute blazes in flames. 
+You would have surely died if you had stayed in there.
+You feel an unexpected pain: a piece of wood pierced your left leg, causing a deep wound that keeps bleeding. 
+Still dazzled and half deafened by the lighting, you see a figure on a distant hill dancing wildly under the storm rains. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.